docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md

```diff
@@ -0,0 +1,210 @@
+
+Most of the distribution images available on Linode use the upstream distribution-supplied kernel by default.
+
+## Determining Which Kernel Is Installed
+
+There are may ways you can determine which kernel version is installed on your Linux system. The following instructions cover the most common methods:
+
+1.  Log in to the Linode through either [SSH](/docs/guides/connect-to-server-over-ssh/) or [Lish](/docs/guides/using-the-linode-shell-lish/).
+
+1.  Run one of the following commands to display the kernel version:
+
+    - `uname -r`: Outputs a single line with just the kernel version.
+    - `hostnamectl`: Outputs an organized list of system details, including the kernel version. Can only be used on distributions with [systemd](https://en.wikipedia.org/wiki/Systemd) (Debian, Ubuntu, CentOS, openSUSE, and more).
+    - `cat /proc/version`: Outputs a string that contains the kernel version and release details.
+
+The kernel version should be a string that's similar to `5.4.0-80-generic`. In this example, the kernel ends with `generic`, which is common in Ubuntu systems to designate that this is an upstream kernel. If the kernel ends with something similar to `linode123`, a Linode kernel is being used and the string exactly which Linode kernel.
+
+## Viewing and Modifying the Kernel in the Cloud Manager
+
+You can use the Cloud Manager to view the type of kernel you are using. If you are using a Linode kernel (not an upstream kernel), you can also view the kernel version.
+
+{{< note >}}
+These instructions may not accurately reflect the actual kernel version installed. This is especially true if the Linode hasn't been rebooted in a while or wasn't rebooted after a change to the Linode's Configuration. To determine the actual kernel version installed on your system, see [Determining Which Kernel Is Installed](#determining-which-kernel-is-installed).
+{{</ note >}}
+
+1.  Navigate to your Linode in the [Cloud Manager](https://cloud.linode.com) and click the **Configuration** tab.
+
+1. Find your current *Configuration* and click the corresponding **Edit** link. This may be located within the **More Options Ellipses** dropdown menu.
+
+1.  Scroll to the *Boot Settings* section.
+
+1.  Review the **Kernel** dropdown menu selection.
+
+    - **Latest 64 bit**: Uses the latest 64-bit Linode kernel at the time the Linode boots/reboots. This was the default for most 64-bit distributions prior to August 2018.
+    - **Latest 32 bit**: Uses the latest 32-bit Linode kernel at the time the Linode boots/reboots. This was the default for most 32-bit distributions prior to August 2018.
+    - **Direct Disk**: Instead of a Linux Kernel, this uses the MBE (Master Boot Record) of the primary disk*.
```

Comment:
I don't think you meant to have the `*` at the end of the sentence?

Reply:
Actually, this `*` is referring to the note at the end of the list. Going to mark this as resolved but let me know if I should do this a different way.